### PR TITLE
Add videos Prague 2022

### PR DESCRIPTION
**Pull request by mxsasha**
----
:books: Documentation preview :books:: https://writethedocs-www--1821.org.readthedocs.build/en/1821/

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
Looks good to me. 

(unrelated, looks like we never did this for Portland 2022?)

**Comment by mxsasha** [comment]
> (unrelated, looks like we never did this for Portland 2022?)

Added too, and also updated the weird uppercase region names for all conferences.

**Review by ericholscher — APPROVED** [review]
Love it!

File: docs/_ext/videos.py [hunk 1]
```diff
@@ -153,3 +155,7 @@ def _make_context(self, data, config, env):
         context = super()._make_context(data, config, env)
         context['key'] = self.options.get('key')
         return context
+
+
+def format_region(region):
+    return region.title() if len(region) > 2 else region.upper()
```
Comment (ericholscher) [inline docs/_ext/videos.py:161]: Clever :)

**Comment by plaindocs** [comment]
Nice one!

**Pull request merged**